src/lib/server/endpoints/aws/endpointAws.ts

```diff
@@ -8,32 +8,45 @@ export const endpointAwsParametersSchema = z.object({
 	model: z.any(),
 	type: z.literal("aws"),
 	url: z.string().url(),
-	accessKey: z.string().min(1),
-	secretKey: z.string().min(1),
+	accessKey: z
+		.string({
+			description:
+				"An AWS Access Key ID. If not provided, the default AWS identity resolution will be used",
+		})
+		.min(1)
+		.optional(),
+	secretKey: z
+		.string({
+			description:
+				"An AWS Access Key Secret. If not provided, the default AWS identity resolution will be used",
+		})
+		.min(1)
+		.optional(),
 	sessionToken: z.string().optional(),
 	service: z.union([z.literal("sagemaker"), z.literal("lambda")]).default("sagemaker"),
-	region: z.string().optional(),
```

Comment:
making the `region` required could be a breaking change for some existing configs no? what is the reasoning for this change? 👀

Reply:
Great catch, I did not mean to commit this -- just an artifact from my testing 🌵

Re-added the .optional()